build.csx

```diff
@@ -183,7 +186,7 @@ targets.Add(
         var version = GetVersion();
         foreach (var nuspec in nuspecs)
         {
-            Cmd(nuget, $"pack {nuspec} -Version {version} -OutputDirectory {outputDirectory} -NoPackageAnalysis");
+            Cmd("nuget", $"pack {nuspec} -Version {version} -OutputDirectory {outputDirectory} -NoPackageAnalysis");

```

Comment:
Why this change? I see we still rely on the `.nuget/nuget.exe` elsewhere. Is there a way to rely on just one?